mdal/frmts/mdal_ugrid.cpp

```diff
@@ -74,36 +71,100 @@ MDAL::CFDimensions MDAL::DriverUgrid::populateDimensions( )
   size_t count;
   int ncid;
 
-  mMesh1dName = findMeshName( 1, true ); // optional, may not be present
-  mMesh2dName = findMeshName( 2, false ); // force
+  /* getting mesh name */
+  std::vector<std::string> mAllMeshesNames = findMeshesNames();
```

Comment:
prefix `m` is used for class  member variables, use `meshesNames`

Reply:
It is a member variable

Reply:
it is not, if it would be member variable, it would be without `  std::vector<std::string>` part

Reply:
You are right, my bad, first it was a local variable, but then I refactored it to be an attribute. However, I left here a declaration.. I think It can help having all mesh names loaded, e. g. in `ignoreNetCDFVariables` we need to loop over all mesh names, or with the new API - selecting specific mesh by name.